### DISCUSSION TITLE
List view

### DISCUSSION THREAD

**Discussion by InnovusSollertia** [community/community#17364] [Other Feature Feedback, Questions, & Ideas] [answered]
Meaning: You create a list with favorite projects (forks, gists, etc.), when you add a project to the list, this project is automatically moved to the created list, and is not shown simultaneously in the created list and the general list of favorite projects

**Comment by fuadmertyu** [comment, marked as answer]
> Meaning: You create a list with favorite projects (forks, gists, etc.), when you add a project to the list, this project is automatically moved to the created list, and is not shown simultaneously in the created list and the general list of favorite projects

**Reply by Alilimucci** [reply]
Ok
